azext_iot/models/parsers.py

```diff
@@ -0,0 +1,252 @@
+import random
+import re
+import json
+
+from knack.log import get_logger
+from uamqp.message import Message
+from azext_iot.common.utility import parse_entity, unicode_binary_map
+
+SUPPORTED_ENCODINGS = ["utf-8"]
+DEVICE_ID_IDENTIFIER = b"iothub-connection-device-id"
+INTERFACE_NAME_IDENTIFIER = b"iothub-interface-name"
+random.seed(0)
+
+
+class Event3Parser(object):
```

Comment:
Good idea to encapsulate the message stream processing in class like `Event3Parser`.